test/functional/wallet_encryption.py

```diff
@@ -64,14 +64,15 @@ def run_test(self):
         assert_raises_rpc_error(-8, "Timeout cannot be negative.", self.nodes[0].walletpassphrase, passphrase2, -10)
         # Check the timeout
         # Check a time less than the limit
-        expected_time = int(time.time()) + (1 << 30) - 600
-        self.nodes[0].walletpassphrase(passphrase2, (1 << 30) - 600)
+        max_value = 100000000
+        expected_time = int(time.time()) + max_value - 600
+        self.nodes[0].walletpassphrase(passphrase2, max_value - 600)
         actual_time = self.nodes[0].getwalletinfo()['unlocked_until']
         assert_greater_than_or_equal(actual_time, expected_time)
         assert_greater_than(expected_time + 5, actual_time) # 5 second buffer
         # Check a time greater than the limit
-        expected_time = int(time.time()) + (1 << 30) - 1
-        self.nodes[0].walletpassphrase(passphrase2, (1 << 33))
+        expected_time = int(time.time()) + max_value - 1
+        self.nodes[0].walletpassphrase(passphrase2, max_value + 1)
```

Comment:
Might want to be off by more than `1`, since we have a "buffer" of 5? Maybe try off by 1000 or something?